settings.py

```diff
@@ -5,6 +5,7 @@
 # Time (in seconds) to wait retrieve box from storehouse
 BOX_RESTORE_TIMER = 0.1
 ENABLE_CONSISTENCY_CHECK = True
+ENABLE_BARRIER_REQUEST = True
```

Comment:
If figured some users might have some switches where barrier isn't correctly supported (and might crash the switch?), so I added this option to enable or disable.

Reply:
good catch! it is always good to have ways of disabling features like this. We never know.